Mathlib/Analysis/Normed/Affine/Isometry.lean

```diff
@@ -171,6 +172,23 @@ theorem diam_image (s : Set P) : Metric.diam (f '' s) = Metric.diam s :=
 theorem diam_range : Metric.diam (range f) = Metric.diam (univ : Set P) :=
   f.isometry.diam_range
 
+/-- Interpret an affine isometry as a continuous affine map. -/
+def toContinuousAffineMap : P →ᴬ[𝕜] P₂ where
+  toAffineMap := f.toAffineMap
+  cont := f.continuous
```

Comment:
I think this sort of thing is best put into a separate PR, declared as a dependency of this one.

Reply:
These changes are now in #31395